docs/quickstarts/python-quickstart.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: docs
+title: Python Quickstart
+---
+
+<h1 class="page-header">Python Quickstart</h1>
+
+<p class="lead">This guide gets you started with gRPC in Python with a simple working example.</p>
+
+<div id="toc"></div>
+
+## Before you begin
+
+### Prerequisites
+
+[versions of pip, any other tools, etc. go here]
+
+If you haven't done so already, install the gRPC runtime and gRPC tools.
+
+### Install gRPC
+
+Install gRPC:
+
+```sh
+  $ pip install grpcio
+```
+
+Or, to install it system wide:
+
+```sh
+  $ sudo pip install grpcio
+```
+
+If you're on Windows, make sure you installed the `pip.exe` component when you
+installed Python. Invoke as above but with `pip.exe` instead of `pip` (you may
+also need to invoke from a `cmd.exe` run as administrator):
+
+```sh
+  $ pip.exe install grpcio
+```
+
+### Install gRPC tools
+
+Python's gRPC tools include the protocol buffer compiler `protoc` and the special plugin for generating server and client code from `.proto` service definitions. For the first part of our quickstart example, we've already generated the server and client stubs from [helloworld.proto](https://github.com/grpc/grpc/tree/{{ site.data.config.grpc_release_branch }}/examples/protos/helloworld.proto), but you'll need the tools for the rest of our quickstart, as well as later tutorials and your own projects.
+
+To install gRPC tools, run:
+
+```sh
+pip install grpcio-tools
+
+```
+
+## Download the example
+
+You'll need a local copy of the example code to work through this quickstart. Download the example code from our Github repository:
+
+```sh
+  $ # Clone the repository to get the example code:
+  $ git clone https://github.com/grpc/grpc
+  $ # Navigate to the "hello, world" Python example:
+  $ cd grpc/examples/python/helloworld
+  ```
```

Comment:
Does this triple-tick need to be without whitespace on its left? When I look at [this rendering](https://github.com/LisaFC/grpc.github.io/blob/96acfeb2839c10c17b54febe61f84815f1f71c4e/docs/quickstarts/python-quickstart.md) the content looks fine but when I look at [this rendering](http://lcarey0-l.dub.corp.google.com:4000/docs/quickstarts/python-quickstart.html) this area of the page is all screwed up.